spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/reader/RowReaderFactory.scala

```diff
@@ -29,25 +29,24 @@ trait ThisRowReaderAsFactory[T] extends RowReaderFactory[T] {
 trait LowPriorityRowReaderFactoryImplicits {
 
   trait IsSingleColumnType[T]
-  implicit def isSingleColumnType[T](implicit ev1: TypeConverter[T], ev2: T IsNotSubclassOf (_, _)): IsSingleColumnType[T] = null
-
-  implicit def classBasedRowReaderFactory[R <: Serializable]
-      (implicit tt: TypeTag[R], cm: ColumnMapper[R], ev: R IsNotSubclassOf (_, _), ev2: R DoesntHaveImplicit IsSingleColumnType[R]): RowReaderFactory[R]  =
+  implicit def isSingleColumnType[T](
+    implicit
+      ev1: TypeConverter[T],
+      ev2: T IsNotSubclassOf (_, _),
+      ev3: T IsNotSubclassOf (_, _, _)): IsSingleColumnType[T] = null
```

Comment:
This is funny because, this result is never actually used - but the implicit might be called, so throwng an exception is not a solution. I'd like to know if we can use something else here instead of null, but nothing comes to my mind.